dashboard/db/migrate/20241212205120_add_foreign_key_from_aichat_events_to_aichat_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddForeignKeyFromAichatEventsToAichatRequests < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :aichat_events, :request, index: true, foreign_key: {to_table: :aichat_requests}, null: true
```

Comment:
Sorry - should've included this question in my earlier review. The table that is being referenced is `aichat_requests` so wondering if [the convention](https://edgeguides.rubyonrails.org/active_record_migrations.html#references) is to use `:aichat_request` instead of `:request` here? Or does the column name not really matter?
And I think an index is already added for this foreign key.

Reply:
That seemed kinda verbose so I went with :request so it's named requestId. I looked at previous migrations to see if there was a precedence for us naming things differently than the default/convention and it looks like we have done this in the past, so I went ahead. I don't have a super strong opinion though if you think using the full name would be better!

Reply:
Sounds good! Thanks for looking and seeing there's precedence for this. The only wonder I have is that `request` is a bit overloaded in meaning. Don't have a strong feeling on this so curious what @sanchitmalhotra126 thinks.

Reply:
Also don't have strong feelings :) in favor of keeping things simple, this makes sense to me!

Reply:
To be clear- which makes sense? as-is or using the simpler syntax to follow the convention?

Reply:
Oh, I meant using `:request` and calling the field `requestId` (as opposed to `:aichat_request`) - that's the simpler version right?

Reply:
Thanks to you both for the discussion - SGTM!